base/applications/notepad/lang/sl-SI.rc

```diff
@@ -25,12 +25,12 @@ BEGIN
     POPUP "&Datoteka"
     BEGIN
         MENUITEM "&Nova\tCtrl+N", CMD_NEW
-        MENUITEM "New &Window\tCtrl+Shift+N", CMD_NEW_WINDOW
+        MENUITEM "Novo O&kno\tCtrl+Shift+N", CMD_NEW_WINDOW
```

Comment:
`&` denominates the hotkey to press, so here it doesn't match the hotkey in the rest of the string.

Reply:
And Okno should be lowercase.

Reply:
the non-matching keys is still a problem

Reply:
> the non-matching keys is still a problem

What do you mean by this?

Reply:
The mnemonic key (`&`) could be different from the hotkey combination, it's totally fine.